src/spryker/harness/attributes/docker.yml

```
@@ -3,6 +3,7 @@ attributes:
     php-base:
       environment:
         APPLICATION_ENV: = @('spryker.mode')
+        DEVELOPMENT_CONSOLE_COMMANDS: "= @('app.build') == 'dynamic' ? 1 : 0"
```

```suggestion
        DEVELOPMENT_CONSOLE_COMMANDS: "= @('app.build') == 'dynamic' ? '1' : '0'"
```
environment variables are always strings